docs/index.rst

```diff
@@ -25,19 +25,6 @@ listed below to find out more.
    :maxdepth: 1
    :caption: API
 
-   topostats.filters
-   topostats.grains
-   topostats.grainstats
-   topostats.io
-   topostats.plotting
-   topostats.plottingfuncs
-   topostats.run_topostats
-   topostats.theme
-   topostats.thresholds
-   topostats.tracing
-   topostats.utils
-   topostats.validation
```

Comment:
Just interested - I understood these as telling GitHub Actions which pages to render on the GitHub Pages site? I'm guessing you're changing how these are selected?

Reply:
The [sphinx-autoapi](https://sphinx-autoapi.readthedocs.io/en/latest/) package now handles this, it sticks a link into the API reference and I figured since most users won't be reading that removing the explicit links from the front-page would do no harm, those interested can still find them.